### ISSUE TITLE
Refresh button?

### ISSUE THREAD

**Issue by fairyaksh** [fac20/Week3-ToDoList-DFJL#39]
- only a suggestion! in case user wants to restart w/ clean template ✨ 
- aware you guys have a delete button but maybe it will make the job easier to have another button to delete all in one go?

**Comment by JoshDAO** [comment]
Good idea! We had loads of things on our stretch goals but we didn't think of this.
